release/script_release.py

```diff
@@ -364,12 +398,10 @@ def release_the_kraken(self):
 if __name__ == '__main__':
 
     if len(argv) < 2:
-        print("mandatory argument: {major|minor|hotfix}")
-        print("possible additional argument: remote (default is canalTP)")
+        print("mandatory argument: {regular|hotfix}")
```

Comment:
I would rather keep the option to perform a `major` and `minor` release in case something goes bad with `regular`.
This would give us more control over the release process, and a backup plan if something goes weird on the release day.

Reply:
It's doable.
The only concern I have about that is releasing a major version and endup with `major number > data_version`. But it can be tested and release aborted in that case?